app/presenters/provider_interface/application_choice_presenter.rb

```diff
@@ -1,24 +1,11 @@
 module ProviderInterface
-  class ApplicationChoicePresenter
-    def initialize(application_choice)
-      @application_choice = application_choice
-      @application_form = application_choice.application_form
-    end
-
-    def id
-      application_choice.id
-    end
-
-    def status
-      application_choice.status
-    end
-
+  class ApplicationChoicePresenter < SimpleDelegator
     def status_tag_text
-      application_choice.status.humanize.titleize
+      status.humanize.titleize
```

Comment:
I'm not following how this works, how does it know where `status` is defined?

Reply:
It's inherited from `< SimpleDelegator`, it causes method missing to resolve everything to the object given to the constructor - @duncanjbrown